### PR TITLE
Update order type dropdown UI

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Updates Order Type dropdown from https://github.com/cowprotocol/cowswap/pull/2275

Screenshot 2023-04-07 at 11 48 32

Screenshot 2023-04-07 at 11 48 49

# To test
- go to limit orders
- turn on expert mode
- you should see order type option

### CODE REVIEW

File: src/cow-react/modules/limitOrders/pure/OrderType/styled.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  margin-left: 5px;
+  width: 8px;
+  transition: all 0.15s ease-in;
+
+  &.expanded {
+    transform: rotate(180deg);
+  }
+`
+
+export const LabelText = styled('span')`
```
Comment (alfetopito) [inline src/cow-react/modules/limitOrders/pure/OrderType/styled.tsx:33]: Nitpick:

```suggestion
export const LabelText = styled.span`
```

File: src/cow-react/modules/limitOrders/pure/OrderType/index.tsx [hunk 2]
```diff
@@ -45,61 +43,25 @@ function OrderTypePicker({ isPartiallyFillable, partiallyFillableOverride }: Ord
...
+              <styledEl.LabelText>{labelText}</styledEl.LabelText>
+              <styledEl.StyledSVG
+                src={IMAGE_CARET_DOWN}
+                description="dropdown icon"
+                className={isExpanded ? 'expanded' : ''}
+              />
+            </styledEl.StyledMenuButton>
+
+            <styledEl.StyledMenuList portal={false}>
+              <styledEl.StyledMenuHeader>Select order type</styledEl.StyledMenuHeader>
```
Comment (alfetopito) [inline src/cow-react/modules/limitOrders/pure/OrderType/index.tsx:59]: Not sure we need a header.
It's just a single element in the list, and I assume it's self explanatory.

**Review by alfetopito — APPROVED** [review]
Thank you!!!

Some minor comments in the code

File: src/cow-react/modules/limitOrders/pure/OrderType/index.tsx [hunk 3]
```diff
@@ -45,61 +43,25 @@ function OrderTypePicker({ isPartiallyFillable, partiallyFillableOverride }: Ord
 
   return (
     <Menu>
-      {({ isExpanded }) => (
+      {({ isExpanded }: { isExpanded: any }) => (
```
Comment (shoom3301) [inline src/cow-react/modules/limitOrders/pure/OrderType/index.tsx:46]:
```suggestion
      {({ isExpanded }: { isExpanded: boolean }) => (
```

File: src/cow-react/modules/limitOrders/pure/OrderType/index.tsx (hunk 2 above)
Comment (anxolin) [reply src/cow-react/modules/limitOrders/pure/OrderType/index.tsx:59]: Same thought

**Review by anxolin — COMMENTED** [review]
It works, but still the styles feel a bit wrong.

I think we don't need the header, and we shouldn't remove one of the options because it is selected. This is not the normal behaviour for a SELECT input

image

I think the select should also stand out a bit more. It feels like any other text field, but is not.

I think is not super coherent with the selector we have for the date. I think this one works fine, why changing? 

image

File: src/cow-react/modules/limitOrders/pure/OrderType/index.tsx (hunk 2 above)
Comment (nenadV91) [reply src/cow-react/modules/limitOrders/pure/OrderType/index.tsx:59]: Ok, will remove it then

**Comment by nenadV91** [comment]
@anxolin 
Updated the UI 
Screenshot 2023-04-10 at 16 15 06
Screenshot 2023-04-10 at 16 16 00

**Review by anxolin — APPROVED** [review]
much better

**Review by elena-zh — APPROVED** [review]
Great job!

**Pull request merged**